lib/ex_debug_toolbar/collector/instrumentation_collector.ex

```diff
@@ -0,0 +1,14 @@
+defmodule ExDebugToolbar.Collector.InstrumentationCollector do
+  alias ExDebugToolbar.Toolbar
+
+  def ex_debug_toolbar_process_request_id(:start, _, _), do: :ok
+  def ex_debug_toolbar_process_request_id(:stop, _, _) do
+    Toolbar.start_request
+    Toolbar.start_event("request")
+  end
+
+  def ex_debug_toolbar(:start, _, _), do: :ok
+  def ex_debug_toolbar(:stop, _, _) do
+    Toolbar.finish_event("request")
```

Comment:
quick question before getting into the review. You are now wrapping the plug that generates the id.
Looks really asymmetrical. you do nothing in `ex_debug_toolbar(:start` nor in `ex_debug_toolbar_process_request_id(:start`

from an outsider point of view, the id should be generated in the `ex_debug_toolbar(:start`, and the request event started.
then, it should be finished in the `ex_debug_toolbar(:stop`
isn't that possible? that way, you don't need anymore the ex_debug_toolbar_process_request_id